CHANGELOG.md

```diff
@@ -27,6 +27,31 @@ This is an example of a brief overview of the _Major_ or _Minor_ version changes
 
 ---
 
+## [10.0] MAJOR
+
+### Changes / Additions
+
+- Fade classes added, which work via pseudo elements and support using multiple at a time
+- FDS Button now has support for adding margins.
+
+### **BREAKING CHANGES**
+
+#### Component reorganization
+- bgFade classes are removed
+
+#### Double and Half spacing removed
+- classes, css variables, and js variables all updated
+- upgrade guide: 
+  - `half` (8px) maps to `s` (8px)
+  - `double` (32px) maps to `xl` (32px)
+- Examples:
+  - CSS Variable: { margin: var(--space-half) } becomes { margin: var(--space-half) }
```

Comment:
think this should be { margin: var(--space-s) }

Reply:
lol good catch, thanks.

Reply:
fixed